api/v1beta1/cindervolume_types.go

```diff
@@ -17,37 +17,80 @@ limitations under the License.
 package v1beta1
 
 import (
+	condition "github.com/openstack-k8s-operators/lib-common/modules/common/condition"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // CinderVolumeSpec defines the desired state of CinderVolume
 type CinderVolumeSpec struct {
-	// CR name of managing controller object to identify the config maps
-	ManagingCrName string `json:"managingCrName,omitempty"`
-	// Cinder Volume storage node selector
-	NodeSelectorRoleName string `json:"nodeSelectorRoleName,omitempty"`
-	// Cinder Database Hostname String
-	DatabaseHostname string `json:"databaseHostname,omitempty"`
-	// Cinder Volume Container Image URL
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=cinder
+	// ServiceUser - optional username used for this service to register in cinder
+	ServiceUser string `json:"serviceUser"`
+
+	// +kubebuilder:validation:Optional
+	// ContainerImage - Cinder API Container Image URL
 	ContainerImage string `json:"containerImage,omitempty"`
-	// Cinder Volume Replicas
+
+	// +kubebuilder:validation:Required
+	// Replicas - Cinder API Replicas
```

Comment:
Cinder Volume Replicas

Reply:
should default to 1 as it is driver depended if multiple volume service are supported for a backend driver